packages/preview-middleware-client/src/adp/controllers/ControllerExtension.controller.ts

```diff
@@ -243,8 +252,9 @@ export default class ControllerExtension extends BaseDialog {
             const data = await getExistingController(controllerName);
             return data;
         } catch (e) {
-            MessageToast.show(e.message, { duration: 5000 });
-            throw new Error(e.message);
+            const error = getError(e);
```

Comment:
Why don't you use `handleError` here?

Reply:
Fixed in [39505b0](https://github.com/SAP/open-ux-tools/pull/1918/commits/39505b01b7d9cb4ccdc43b1ab21387e06780dc7a)